aspen/http/request.py

```diff
@@ -226,6 +226,13 @@ def __new__(cls, method=b'GET', uri=b'/', server_software=b'',
 
             raise Response(400, "Request is undecodable. "
                                 "(%s:%d)" % (filename, frame.f_lineno))
+
+        # set up aliases
```

Comment:
Are we sure these are lazy enough?  None of the RHS can/will change after the 'aliases' are set?